pisek/config/task_config.py

```diff
@@ -226,6 +237,18 @@ def validate_model(self):
                 {"task_type": self.task_type, "out_check": self.out_check},
             )
 
+        if (self.tokens_float_abs_error == None) != (
```

Comment:
Nit: Pyhton style prefers ``is None`` instead of ``== None``.

Reply:
Fixed.